### ISSUE TITLE
need a ui

### ISSUE THREAD

**Issue by xycloud** [dataux/dataux#54]
need a ui for this awesomeness

**Comment by araddon** [comment]
Thanks for the input!   I have been using http://www.metabase.com/ and just MySQL console tools.  would love thoughts on what specific areas need a UI?   I have also been working on creating more support for traditional `SQL CREATE` statements for traditional sql schema management.

**Comment by xycloud** [comment]
cool, could you provide a docker image for friendly try out?

**Comment by araddon** [comment]
Absolutely, I have some docker ready images, and have been converting all of the backend sources to support sql `CREATE` statements for on-the-fly registering backend sources.  Ill go through and update doc on home page to reflect more.

https://github.com/dataux/dataux/tree/master/backends/files
